okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/OkHttpClients.java

```diff
@@ -85,7 +85,13 @@
     private static final Dispatcher dispatcher;
 
     /** Shared connection pool. */
-    private static final ConnectionPool connectionPool = new ConnectionPool(100, 10, TimeUnit.MINUTES);
+    private static final ConnectionPool connectionPool = new ConnectionPool(
+            1000,
+            // Most servers use a one minute keepalive for idle connections, by using a shorter keepalive on
+            // clients we can avoid race conditions where the attempts to reuse a connection as the server
+            // closes it, resulting in unnecessary I/O exceptions and retrial.
+            55,
```

Comment:
I was looking for the code for this WC, but isn't that 1 minute timeout for worker threads?

Reply:
We don't override the default, though we probably should override it with itself for discoverability.

https://github.com/undertow-io/undertow/blob/062e01fbbf23195cde76a9d59e274fccd534ae00/core/src/main/java/io/undertow/UndertowOptions.java#L75-L78

https://github.com/undertow-io/undertow/blob/8b243e1e75812bcef15fc1ec4bf0a837ec8ce4bb/core/src/main/java/io/undertow/Undertow.java#L144-L147